charts/terrarium/templates/deployment_secret.yaml

```diff
@@ -7,3 +7,4 @@ data:
   AWS_ACCESS_KEY_ID: {{ .Values.aws_access_key_id }}
   AWS_SECRET_ACCESS_KEY: {{ .Values.aws_access_key_secret }}
   AWS_DEFAULT_REGION: {{ .Values.aws_region }}
+  WEBHOOK: {{ .Values.webhook }}
```

Comment:
Does the webhook really need to be a secret?

Reply:
Since we already had an implementation in place for secrets I decided to use this approach for now. I tried to pass it as a general Environment variable but faced a few issues so I stuck to this approach.